src/components/TradeWidget/index.tsx

```diff
@@ -237,46 +412,78 @@ const TradeWidget: React.FC = () => {
             onSelectChange={onSelectChangeFactory(setSellToken, receiveToken)}
             inputId={sellInputId}
             isDisabled={isSubmitting}
             validateMaxAmount
             tabIndex={1}
           />
           <IconWrapper onClick={swapTokens}>
-            <FontAwesomeIcon icon={faExchangeAlt} rotation={90} size="2x" />
+            <img src={switchTokenPair} />
           </IconWrapper>
           <TokenRow
             selectedToken={receiveToken}
             tokens={tokens}
             balance={receiveTokenBalance}
-            selectLabel="receive"
+            selectLabel="Receive at least"
             onSelectChange={onSelectChangeFactory(setReceiveToken, sellToken)}
             inputId={receiveInputId}
             isDisabled={isSubmitting}
             tabIndex={2}
           />
+          {/* Refactor these price input fields */}
+          <PriceWrapper>
+            <strong>Min. sell price</strong>
+            <PriceInputBox>
+              <label>
+                <input placeholder="0" value="146.666" type="text" required />
+                <small>WETH/DAI</small>
+              </label>
+            </PriceInputBox>
+            <PriceInputBox>
+              <label>
+                <input placeholder="0" value="0.00682" type="text" required />
+                <small>DAI/WETH</small>
+              </label>
+            </PriceInputBox>
+          </PriceWrapper>
+          {/* Refactor these price input fields */}
           <OrderValidity
             inputId={validUntilId}
             isDisabled={isSubmitting}
             isUnlimited={unlimited}
             setUnlimited={setUnlimited}
             tabIndex={3}
           />
-          <OrderDetails
+          {/* <OrderDetails
             sellAmount={watch(sellInputId)}
             sellTokenName={safeTokenName(sellToken)}
             receiveAmount={watch(receiveInputId)}
             receiveTokenName={safeTokenName(receiveToken)}
             validUntil={watch(validUntilId)}
-          />
-          <SubmitButton type="submit" disabled={!methods.formState.isValid || isSubmitting} tabIndex={5}>
-            <FontAwesomeIcon icon={isSubmitting ? faSpinner : faPaperPlane} size="lg" spin={isSubmitting} />{' '}
-            {sameToken ? 'Please select different tokens' : 'Send limit order'}
+          />{' '} */}
+          <p>This order might be partially filled.</p>
+          <SubmitButton
+            data-text="This order might be partially filled."
+            type="submit"
+            disabled={!methods.formState.isValid || isSubmitting}
+            tabIndex={5}
+          >
+            {isSubmitting && <FontAwesomeIcon icon={faSpinner} size="lg" spin={isSubmitting} />}{' '}
+            {sameToken ? 'Please select different tokens' : 'Submit limit order'}
           </SubmitButton>
         </WrappedForm>
       </FormContext>
+      <OrdersPanel>
+        {/* Toggle panel visibility (arrow) */}
+        <button></button>
```

Comment:
```suggestion
        <button/>
```

Emptybutton? Wouldn't it be better to add whatever is displayed in css (`:after or background-image` or whatever) into code?

Reply:
cc @biocom